Octokit.Generators/Generator.cs

```diff
@@ -0,0 +1,29 @@
+using System.Threading.Tasks;
+
+namespace Octokit.Generators
+{
+  /// <summary>
+  /// Provides an entry point for code generation of various types.
+  /// </summary>
+  /// <remarks>
+  /// The backing source for generation will either be the source files in this repo or
+  /// the OpenAPI Descriptions from the GitHub REST API: https://github.com/github/rest-api-description
+  /// </remarks>
+  class Generator
+    {
+
+        static void Main(string[] args)
```

Comment:
Async Task Main?

Reply:
I didn't see a good reason to make that jump right now... it wouldn't provide any value IMO. I could go either way.

Reply:
👍 I'm easy either way.